File: src/java/eu/sig/training/ch03/withmapandtypes/FrenchFlag.java
package eu.sig.training.ch03.withmapandtypes;

import java.awt.Color;
import java.util.Arrays;
import java.util.List;


public class FrenchFlag implements Flag {
    public List<Color> getColors() {
        return Arrays.asList(Color.BLUE, Color.WHITE, Color.RED);
    }
}

File: src/csharp/eu/sig/training/ch03/withmapandtypes/FrenchFlag.cs
using System.Collections.Generic;
using System.Drawing;

namespace eu.sig.training.ch03.withmapandtypes
{

    public class FrenchFlag : IFlag
    {
        public IList<Color> Colors
        {
            get
            {
                return new List<Color> { Color.Blue, Color.White, Color.Red };
            }
        }
    }

}
